collector/exporter.go

```diff
@@ -68,6 +68,8 @@ type Collect struct {
 	UserStat                bool
 	ClientStat              bool
 	TableStat               bool
+       InnodbCmp               bool
```

Comment:
I think this indenting is off, please run `make format`.

Reply:
Done.